README.md

```diff
@@ -89,7 +89,7 @@ are used, these must be in `images` folders next to the .md file referencing the
 To add version information and extra keywords, [every document can be annotated using YAML](Contribute/adding-metadata.md). 
 
 # Multi version documentation
-With the introduction every new version of Umbraco new features are introduced.  This means that not every document will work for your possibly older version.
+With the introduction, every new version of Umbraco new features is introduced.  This means that not every document will work for your possibly older version.
```

Comment:
You are correct that this sentence didn't make any sense, I don't think your edit made it any clearer though - you are very welcome to rewrite the sentence but for now I will decline this PR.